tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
-    "target": "ES2021",
+    "target": "es2021",
```

Comment:
It's not something you should ever change, but is there a reason you did? (Just curious)

Reply:
Some library (Maybe Vite) was checking tsconfig's target is `"es2021"` without normalizing letter cases. So I need to change it into lowercase. There's no difference in tsc between uppercase and lowercase. 😄